Command/NotifyDeploymentCommand.php

```
@@ -73,17 +73,38 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $newrelic = $this->newrelic;
 
-        $response = $this->performRequest($newrelic->getApiKey(), $this->createPayload($newrelic, $input));
-
-        if (empty($response)) {
-            $output->writeLn("<error>Deployment not recorded: Did not understand response</error>");
-        } elseif (!in_array($response['status'], array(200, 201))) {
-            $output->writeLn(sprintf('<error>Deployment not recorded. Received "%d - %s"</error>', $response['status'], $response['error']));
-        } else {
-            $output->writeLn(sprintf('<info>Recorded deployment to "%s" (%s)</info>', $newrelic->getName(), ($input->getOption('description') ? $input->getOption('description') : date('r'))));
+        $appNames = $newrelic->getDeploymentNames();
+
+        if (!$appNames) {
+            $output->writeLn("<error>No deployment application configured.</error>");
```

you should return an error exit code. So callee can stop their processus.